core/src/nativeMain/kotlin/com/powersync/sqlite/Database.kt

```diff
@@ -52,22 +52,26 @@ internal class Database(
             Statement(sql, ptr, stmtPtr.value!!)
         }
 
-    fun loadExtension(
+    public fun loadExtension(
```

Comment:
If the intention is to call this method from Swift, should it have a `@Throws` annotation?